### ISSUE TITLE
Releasing next version to Curse

### ISSUE THREAD

**Issue by NielsH** [Immemor/RaidCore#43]
Hi,

About a month ago I released the previous version to Curse. Now, we have added locale support and settings for DS, as well as a bunch of bugfixes.

I want to add a new version to Curse so people can use the new stuff. For that to happen I think at the moment there are 2 bugs that need to be fixed in advance:  
- Boss module not activating. From what I tested, this one will be fixed with https://github.com/NielsH/RaidCore/pull/39  
- Chat messages/datachron messages sometimes(?) not being picked up. I'm not sure if it doesn't work in some modules, or if it doesn't work anywhere, that'll need some additional testing. As mentioned somewhere else, it seems to be an issue at the moment on System Daemons (placing world markers) and Ohmna (only first pillar bar shows) at least.

Once this is fixed I'd like to release to Curse so people can be happy with a new release that has a bunch of new things! Then we can continue again on other/new stuff. That way the people using the addon can also see there is active development and don't complain about slow updates :+1:

**Comment by Immemor** [comment]
Which policy to you have ?

Delivery directly a release version, or an alpha first ? If we provide an alpha, how many time before the next ? 1 week ?

**Comment by NielsH** [comment]
I've tried alpha's for some of my other addons, but very few people use it and generally I found it not to be worth it even trying. I test the major stuff (i.e. no errors/major issues during boss encounters during a reset, no errors clicking through UI) and then just upload it as a new release.

If there are issues that pop up then people will let us know quickly, and if severe enough it's possible to do a quick bugfix release.

**Comment by Immemor** [comment]
One bug found in EpFireWater.

Fix pushed in master.

**Comment by NielsH** [comment]
Cool, I also pushed an issue that prevented settings from working.

I think for next reset it's important to test if modules are starting properly, I think that has been the biggest issue lately, modules either not starting or not loading at all. As well as possibly the ChatDC stuff, though from the fights that I did... that bit did work. After that we can make up a changelog summary and add it to Curse.

What do you think?

**Comment by Immemor** [comment]
I'm testing my branch with low layer reworked. Because I don't trust the current low layer.
Because I have log, and I started to deploy the idea of protected function too.

I was near to found the second bug ^^, with my debug tool. I will try tonight with your fix. I should see this timers on Air/Water encounter.

**Comment by NielsH** [comment]
So having tested GA/DS this week, the only "breaking" bug I see is that OnChatDC still seems iffy, so for Ohmna it only shows the first pillar assignment, but it doesn't show you all the next pillars at all. I also tested with the logger-branch, did that have your fix in it? (referring to https://github.com/NielsH/RaidCore/issues/36#issuecomment-94943773 )

Once that is fixed I think we can release to curse?

**Comment by Immemor** [comment]
I don't do a lot of GA now, but tonight I will do it.

I will have a look.

**Comment by Immemor** [comment]
I tried Ohmna with my branch & in french (I added french translations).

In Log:
- I had the entry **"Id=263799 Unit='Ohmna la Terriphage' InCombat=true"**
- I haven't the **"TrackThisUnit"**

As I reload my ui very often, the module was probably not enable. As this last is based on CreateUnit event, And I haven't "UniteCreate" before "EnteringCombat" in log.

So, that will be next step.

Otherwise, I have all datachron messages in log.
